doc-content/enterprise-only/openshift/operator-deploy-security-proc.adoc

```diff
@@ -30,7 +30,7 @@ If you selected `RH-SSO`, configure RH-SSO authentication:
 If you selected `LDAP`, configure LDAP authentication:
 
 .. In the *LDAP URL* field, enter the LDAP URL.
-.. Configure LDAP parameters that correspond to the settings of the LdapExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.3/html/login_module_reference/login_modules_with_external_identity_store#ldapextended_login_module[LdapExtended Login Module].
+.. Configure LDAP parameters that correspond to the settings of the LdapExtended Login module of {EAP}. For instructions about using these settings, see https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.4/html/login_module_reference/login_modules_with_external_identity_store#ldapextended_login_module[LdapExtended Login Module].
```

Comment:
We do not use this the LdapExtended anymore, we have migrated to elytron, we need to make users aware of this change and maybe refer to the elytron doc.

Reply:
Aha, THIS is interesting and I will search for the appropriate docs.